locales/en/index.yml

```diff
@@ -2457,17 +2459,19 @@ features:
         title: "TMP"
         subtitle: "TMP"
       notFound:
-        title: "TMP"
-        subtitle: "TMP"
+        title: "The certificate does not appear to \nexist: contact support"
```

Comment:
```suggestion
        title: "It seems that the certificate does not \nexist: contact support"
```

Reply:
Here the result:
<img src="https://user-images.githubusercontent.com/11773070/120995661-e581d100-c785-11eb-8f43-513c59140c73.png" width=300>